vuu-ui/packages/vuu-filters/src/filter-clause/useFilterClause.ts

```diff
@@ -160,6 +162,14 @@ export const useFilterClause = ({
     ],
   );
 
+  const handleOpenChange = (open: boolean, closeReason?: OpenChangeReason) => {
+    const isMultiSelect = filterClauseModel.op === 'in';
+    if(!open && isMultiSelect && filterClauseModel.isValid) {
+        filterClauseModel.emit('filterClause', filterClause, true);
```

Comment:
I understand why you need to to this. It feels wrong that an external party calls emit on another object. I feel that emitting an event is something an eventEmitter does to indicate some internal state change .
If I want to understand when an object emits events, I would expect to just have to scan the source code for that object (or ancestors) not scan the entire code base.

I'd suggest the following compromise ; add a `commit` method to the FIlterClauseModel, which just emits the `filterClause` event, then call `filterClauseModel.commit() from your code.

That both keeps the eventEmitter logic internal and also signals that external callers can trigger the emit